### ISSUE TITLE
connection hardware 

### ISSUE THREAD

**Issue by Lynda96** [ttlappalainen/NMEA2000#260]
Hi,
I have a question about  connection hardware  ,Can I read wind data from NMEA2000 by doing this connection ?,
![schémaaneno](https://user-images.githubusercontent.com/93643631/158212508-01688a55-2b5b-45b3-bd6e-76b0fc1a4084.PNG)
 I mean ,can I read data from sensor directly because on DataDispaly2 example the are two Arduino boards connected to NMEA2000 one for sending and the second one for receveing what the one has sent

**Comment by ttlappalainen** [comment]
I do not know what you mean with "read data wind from sensor directly". You need microcontroller on both sides. One runs WindMonitor example, which you develop to read sensor. Other runs DataDisplay example, which you modify to show wind data on display.

**Comment by Lynda96** [comment]
ok now I understand so what i did in the picture is false , I need to connect the sensor to microcontroller for mesuring wind speed for example  and after that I send data to NMEA2000 and receive this data with other microcontroller if i want isn't it?

**Comment by ttlappalainen** [comment]
Yes. You must think NMEA2000 as ethernet - it is a bus to transfer data between devices, which must be cabable to communicate throught the bus.
